content/uiux/concepts/design-thinking/design-thinking.md

```diff
@@ -0,0 +1,32 @@
+---
+Title: 'Design Thinking'
+Description: 'Design thinking is a design methodology that puts people at the center of every process and encourages designers to set aside assumptions.'
+Subjects:
+  - 'Web Design'
+Tags:
+  - 'UI'
+  - 'UX'
+CatalogContent:
+  - 'intro-to-ui-ux'
+  - 'paths/front-end-engineer-career-path'
+---
+
+**Design thinking** is a [design methodology](https://www.codecademy.com/resources/docs/uiux/design-methodologies) that puts people at the center of every process and encourages designers to set aside assumptions. This formal methodology has [developed across multiple disciplines since the 1960s](https://www.interaction-design.org/literature/article/design-thinking-get-a-quick-overview-of-the-history) and is commonly associated with the design and consulting firm [IDEO](https://designthinking.ideo.com/) and the Stanford School of Design (the [d.school](https://dschool.stanford.edu/)).
+
+![A diagram of the five phases of the design thinking process: Empathize, Define, Ideate, Prototype, and Test.](https://static-assets.codecademy.com/Courses/intro-to-ui-and-ux/key-methodologies/design-thinking-5-steps.png)
```

Comment:
@lucwhite Would it make more sense to move this image below the last numbered item on line #24? At its current location, it puts an awkward gap between the new header and the proceeding text.

Reply:
I'm fine with that.